Nos/Views/NoteCard.swift

```diff
@@ -96,14 +78,11 @@ struct NoteCard: View {
                         }
                         BeveledSeparator()
                         HStack(spacing: 0) {
-                            if showReplyCount {
-                                StackedAvatarsView(avatarUrls: replyAvatarURLs, size: 20, border: 0)
-                                    .padding(.trailing, 8)
-                                if let replies = attributedReplies {
-                                    Text(replies)
-                                        .font(.clarity(.medium, textStyle: .subheadline))
-                                        .foregroundColor(Color.secondaryTxt)
-                                }
+                            if repliesDisplayType != .displayNothing {
+                                RepliesLabel(
```

Comment:
Can we put this inside it's own `Button`? It's really nice to see it give feedback when tapped in the `StoryNoteView` and it would be nice to get it here as well.